sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,906 @@
+		# To read in column names
+		data_all = open(filepath)
+		for i, line in enumerate(data_all):
+			if i > 13 :
+				 header.append(line)
+			if line == 'BEGIN DATA\n':
+				data_start = i+1
+				break
+		data_all.close()
+
+		specie = header[-2][header[-2].index(':')+3:header[-2].index('t')-1]
+		header = header[:-2]
+
+		for i in range(len(header)):
+			header[i] = header[i][ header[i].index(":") + 2:] 
+
+		header = ['DateTime'] + header
+
+		
+		#To format column names 
+		for i in range(len(header)-2):
+			header = header + [specie + ' total counts for '+ (header[i+2])[1:20] +' energy range']
+
+		data = ascii.read(filepath, delimiter = "\s", data_start = data_start) 
+	
+		data_modify = []
+		
+		#Storing data columns in recognizable variables
+		year_col = data['col1']
+		day_of_year_col = data['col2']
+		hour_col = data['col3']
+		minutes_col = data['col4']
+		seconds_col = data['col5']
+		
+		#Combining Date, Time columns to make a single datetime.datetime value column 
+		for i in range(len(data)): 
+			date = datetime(year_col[i], 1, 1) + timedelta(int(day_of_year_col[i]) - 1)
+			data_modify.append(datetime(date.year, date.month, date.day, hour_col[i], minutes_col[i], seconds_col[i]))
+	
+		#Adding one DateTime column and removing 5 columns with separated time info
+		data.add_column(Column(data = data_modify, name='col'),1)
+		data.remove_columns(['col{}'.format(i) for i in range(1,6)])
+	
+	
+		#To add the column names in the astropy table object
+		for elem, head_key in enumerate(header):
+			data.rename_column(data.colnames[elem], head_key)        
+	
+		#Converting from astropy.table.Table to pandas.Dataframe
+		# to_pandas() bound method is only available in the latest development build of astropy and 
+		# none of the stable versions include it
+		data = _to_pandas(data)
+	
+		n = len(header)
+		return [OrderedDict(zip(range(n), header)), data]
+
+
+class PLASTICLightCurve(LightCurve):
+
+	"""
+	STEREO PLASTIC LightCurve. Currently the LightCurve supports only Single File Load !!
+
+	Parameters
+	----------
+	timerange: sunpy.time.TimeRange
+		time range for which data is to be downloaded.
+		Example value -  TimeRange('2007-02-14','2014-12-17')   
+
+	stereo_spacecraft: string   
+		Default value - ahead
+		Possible values - ahead, behind    # corresponding to spacecraft location
+
+
+	duration_of_average: astropy units ( Default value - 10*u.min )
+		Possible values - 1*u.min, 10*u.min, 1*u.h      
+		#corresponding to duration over which data is averaged
+
+	Examples
+	--------
+	>>> import os
+	>>> import sunpy.data.test
+	>>> filepath = sunpy.data.test.rootdir
+	>>> from sunpy import lightcurve as lc
+	>>> plastic = lc.PLASTICLightCurve._parse_txt(os.path.join(filepath , 'plastic/STA_L2_PLA_1DMax_1min_20140101_001_V09.txt'))
+	>>> plastic = lc.PLASTICLightCurve(plastic[1],plastic[0])
+	>>> plastic.peek()
+
+	References
+	----------
+	| http://stereo-ssc.nascom.nasa.gov/data/ins_data/plastic/level2/Protons/ASCII/
+
+	"""
```

Comment:
As above, 
 - not `#` lines in the documentation
 - `os.path.join` for directory and filename
 - use the `plot` directive